src/Logary/Configuration_FactoryApi.fs

```diff
@@ -57,14 +65,51 @@ and internal MetricsConfBuilder(conf) =
 
 /// The "main" fluent-config-api type with extension method for configuring
 /// Logary rules as well as configuring specific targets.
-and ConfBuilder(conf) =
+and ConfBuilder(conf, internalLoggingLevel) =
+
+  let specificHieraForSetInternalLog = "^specificHieraForSetInternalLog$"
+
+  let confBuilder conf =
+    ConfBuilder (conf,internalLoggingLevel)
+  
+  let configInternalTargets conf =
+    // find all targets set for internal log through a specific rule,
+    // and remove these (specific rule/target) from logary conf, so normal logging is not affected. 
+    // if there isn't any, use console as default
+
+    let (normalRules, forInternalRules) =
+      conf.rules 
+      |> List.partition (fun r -> 
+            match r with
+            | r when r.hiera.ToString() = specificHieraForSetInternalLog -> false
```

Comment:
here is a trick, in order to identity which targets are set for internal logging.  the specific rule with hiera `specificHieraForSetInternalLog ` is for matching a target through its `rule.target` property. 

not find other good way to do it , maybe we can use some conventions to do it , like:  a target with name match some  hiera is the internal target . but maybe affect some exists code ?